lib/commonConfig.js

```diff
@@ -219,7 +219,7 @@ function processEnvironmentVariables() {
     }
```

Comment:
Maybe an entry in CHANGES_NEXT_RELEASE file about the changes done by this PR should be added?

Reply:
Fixed  a444160